FabricExample/Gemfile

```
@@ -6,3 +6,5 @@ ruby ">= 2.6.10"
 # Exclude problematic versions of cocoapods and activesupport that causes build failures.
 gem 'cocoapods', '>= 1.13', '!= 1.15.0', '!= 1.15.1'
 gem 'activesupport', '>= 6.1.7.5', '!= 7.1.0'
+gem 'xcodeproj', '< 1.26.0'
+gem 'concurrent-ruby', '< 1.3.4'
```

Please add this missing whitespace here
```suggestion
gem 'concurrent-ruby', '< 1.3.4'

```

---

You might want to configure your editor so that it always adds trailing new line to the files.